app/test_main.py

```diff
@@ -1,3 +1,23 @@
-from app.main import get_human_age
+from app.main import get_human_age, convert_to_human
 
-# write your code here
+
+def test_convert_to_human_age() -> None:
+    assert convert_to_human(0, 15, 9, 4) == 0
+    assert convert_to_human(14, 15, 9, 4) == 0
```

Comment:
The test cases for `convert_to_human` with inputs `(0, 15, 9, 4)` and `(14, 15, 9, 4)` both expect the same output `0`. Consider removing one of these to avoid repetition as per the checklist requirement: "Make sure that your tests don't test the same case few times."